packages/cli/src/commands/gsn-relayer-withdraw.ts

```diff
@@ -0,0 +1,44 @@
+import { CommandsLogic, WithdrawOptions } from '../CommandsLogic'
+import { gsnCommander, getKeystorePath, getServerConfig } from '../utils'
+import { createCommandsLogger } from '../CommandsWinstonLogger'
+import { KeyManager } from '@opengsn/relay/dist/KeyManager'
+import { toBN } from 'web3-utils'
+
+const commander = gsnCommander(['f', 'g'])
+  .option('-k, --keystore-path <keystorePath>', 'relay manager keystore directory', '/app/data/gsndata/manager/')
+  .option('-s, --server-config <serverConfig>', 'server config file', '/app/data/config/gsn-relay-config.json')
+  .option('-b, --broadcast', 'broadcast tx after logging it to console', false)
+  .requiredOption('-a, --amount <amount>', 'amount of funds to withdraw to owner address, in wei')
+  .parse(process.argv);
+
+(async () => {
+  const config = getServerConfig(commander.serverConfig)
+  const host = config.ethereumNodeUrl
+  const logger = createCommandsLogger(commander.loglevel)
+  const logic = await new CommandsLogic(host, logger, {}).init()
+  const keystorePath = getKeystorePath(commander.keystorePath)
+  const keyManager = new KeyManager(1, keystorePath)
+
+  const withdrawOptions: WithdrawOptions = {
+    withdrawAmount: toBN(commander.amount),
+    keyManager,
+    config,
+    broadcast: commander.broadcast
+  }
+  console.log('config is', config)
+  console.log('withdrawalAmount is', withdrawOptions.withdrawAmount.toString())
+  console.log('broadcast is', withdrawOptions.broadcast)
+  const result = await logic.withdrawToOwner(withdrawOptions)
```

Comment:
cli should also:
- display current balance (e.g. if no amount is given, or `--list`)
- withdraw all (either with `--amount all` or `--all`)

Reply:
Re displaying balance, I will add though not sure if really needed. If you use this cli, you probably checked the balance already and are supposed to know how much you want to withdraw. Otherwise, this cli is not for you imo.
Re withdraw all - adding a flag to shoot yourself in the leg isn't smart. I disgree. 
If you want to withdraw all, you may as well unstake the server and wait for it to send you all the funds anyway. Withdrawing all balance cannot guarantee the manager still have enough funds both for itself and the worker address. You might need to send the balance back to the manager afterwards. That's not smart.